app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -317,6 +318,12 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
 
             override fun onLoadFinished(loader: Loader<Cursor>, cursor: Cursor?) {
                 adapter.changeCursor(cursor)
+                lifecycleScope.launchWhenResumed {
+                    threadDb.getRecipientForThreadId(threadID)?.let {
+                        println("Refresh pic")
```

Comment:
We may need to remove this one or replace with the `Log` static call

Reply:
Sorry about that, will certainly remove, not sure I wanted a permanent log statement here.

Reply:
No dramas, yeah might be pretty visual enough to not need the log either way :)
The only other thought I had when testing it on my device is that when the image returns it might not be shown to the user here until the thread content updates, which is probably fine :+1: it just meant that joining an open group might download the image a lot quicker than it processes all the first messages and it doesn't update until that point. But it's probably fine to do it this way for now unless it was an issue down the track anyway :tada: